### ISSUE TITLE
lib.es6.d.ts does not include typings for Headers.prototype.entries

### ISSUE THREAD

**Issue by ithinkihaveacat** [Microsoft/TypeScript#14412]
**TypeScript Version:**  2.2.1

**Code**

```ts
$ cat test.ts
const h = new Headers();

h.entries();
$ tsc test.ts
test.ts(3,3): error TS2339: Property 'entries' does not exist on type 'Headers'.
```

**Expected behavior:**

TypeScript does not complain about the `entries` property, since it is [part of the spec](https://developer.mozilla.org/en-US/docs/Web/API/Headers) and implemented by Firefox and Chrome.

**Actual behavior:**

TypeScript complains that the `entries` property does not exist.
